public/js/admin-parts.js

```diff
@@ -8,6 +8,32 @@ $(document).ready(() => {
       description: $('.description').val()
     }
 
-    $.ajax({ url: '/admin/parts', type: 'POST', data: partsData }).done()
+    $.ajax({ url: '/admin/parts', type: 'POST', data: partsData }).done(results => {
+    	if (results.status === 'failure') {
+	$('.error-message').text(results.message)
+	} 
+	else {
+	
+	
+		//appendTeam
+
+		let newPartHTML = [
+			'<li>',
+			'<h1>', partData.partName, '</h1>',
+			'<h1>', partData.stock, '</h1>',
+			'</li>'
+		]
+
+		newPartHTML = newPartHTML.join('')
+
+		$(newPartHTML).appendTo('.team-list')
```

Comment:
So this whole section where you create `newPartHTML` and append it to the parts list - Please move it into a separate function called `appendNewPart` that takes one argument (the parts data). Then replace the old logic with just a call to that function.